src/neo-vm/neo-vm.csproj

```diff
@@ -1,10 +1,11 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <Copyright>2016-2019 The Neo Project</Copyright>
     <AssemblyTitle>Neo.VM</AssemblyTitle>
     <Description>Neo.VM</Description>
-    <Version>3.0.0-preview1</Version>
+    <VersionPrefix>3.0.0</VersionPrefix>
```

Comment:
You really want to use a tool rather than manage this manually.

Reply:
I'm going to use `--version-suffix`.

Reply:
I can tell you from personal experience that managing the version suffix manually will not work out well. 

The problem with most versioning schemes is that they are driven by either by date information, by some build system intrinsic (like the number of previous builds executed in a set amount of time) or both. This makes the version number completely non deterministic.

Nerdbank GitVersioning on the other hand produces deterministic version numbers with no dependency on git tags or branches. So building a specific commit locally and on the build server produce the exact same build version. There are also tools to calculate which commit generated a given version.

Reply:
But we need an incremental version number. Now I'm using the linux timestamp.

https://github.com/neo-project/neo-vm/blob/d358d7e3562877428f9a4eb529e43788dcdf4766/.github/workflows/publish.yml#L22

It generates the package version like this:

```
3.0.0-CI1574448946
```

Reply:
Again, timestamps are not non deterministic. Nerdbank GitVersioning uses a concept called [git height](https://github.com/AArnott/Nerdbank.GitVersioning#what-is-git-height) to give each commit it's own deterministic incremental version number.

Reply:
Now I'm using deterministic version numbers. They are generated by the following command:

```
git rev-list --count HEAD |xargs printf "CI%05d" |xargs dotnet pack -c Debug -o out --include-source --version-suffix
```

Reply:
@devhawk  I agree with you that we could use Nerdbank, but I think this may be outside the scope of this PR.